react/Modal/ModalBackButton.jsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react'
+import cx from 'classnames'
+import { Button } from '../Button'
+import withBreakpoints from '../helpers/withBreakpoints'
+import styles from './styles.styl'
+
+const DumbModalBackButton = props => {
+  const {
+    breakpoints: { isMobile },
+    className,
+    ...rest
+  } = props
+
+  return (
+    <Button
+      icon="previous"
+      iconOnly
```

Comment:
Can you add a label plz ?

Reply:
Did you mean accessibility instead of i18n? Actually since the props are spread, the underlying Button will receive the label that is passed to the ModalBackButton from the app. But I added a default "back" label.

Reply:
yes for accessibility sorry :/ Thanks